source/_integrations/nest.markdown

```diff
@@ -192,6 +192,8 @@ Once your developer account is set up and you have a valid `nest` entry in `conf
     - You can use any publicly known hostname you own
     - As a hack, you can use hosts tricks to temporarily assign a public hostname to your Home Assistant IP address.
 
+- *Error: invalid_client no application name* means the [OAuth Consent Screen](https://console.developers.google.com/apis/credentials/consent) has not been fully configured for the project.  Simply enter the required fields (App Name, Support Email, Developer Email) and leave everything else as default.
```

Comment:
- Remove double space occurrence.
- Removed "Simply" occurrence. It can be condescending for people that don't find it as easy.

```suggestion
- *Error: invalid_client no application name* means the [OAuth Consent Screen](https://console.developers.google.com/apis/credentials/consent) has not been fully configured for the project. Enter the required fields (App Name, Support Email, Developer Email) and leave everything else as default.
```